frontend/packages/console-app/src/components/quick-starts/catalog/QuickStartCatalog.tsx

```diff
@@ -38,30 +49,90 @@ const QuickStartCatalog: React.FC<QuickStartCatalogProps> = ({
   setActiveQuickStart,
 }) => {
   const { t } = useTranslation();
+
+  const [filterText, setFilterText] = React.useState('');
+  const [filterStatus, setFilterStatus] = React.useState([]);
+  const onFilter = (val: string, statusFilters: QuickStartFilteredStatus) => {
+    setFilterText(val.toLowerCase());
+    setFilterStatus(Object.keys(statusFilters).filter((key) => statusFilters[key]));
+  };
+
+  const clearFilters = () => {
```

Comment:
If it's only clearing text filter.
```suggestion
  const clearTextFilter = () => {
```

Reply:
Updated callback to clear all filters.